trview.app/Menus/ViewMenu.cpp

```diff
@@ -133,6 +133,26 @@ namespace trview
                 }
                 break;
             }
+            case ID_APP_VIEW_BACKGROUNDCOLOUR:
+            {
+                static DWORD rgb;
+                static COLORREF custColours[16];
+                CHOOSECOLOR cc;
```

Comment:
Can you set `hwndOwner` to the current window? The picker appears at the top left of the screen instead of near the viewer.
Also maybe you can put the trademark trview blue in as a default custom colour? Then you can go back to it if you like.